website/docs/infrastructure/archive-data.md

```diff
@@ -0,0 +1,160 @@
+# Archiving Data from Decommissioned Projects
+
+## Overview
+
+When a project is decommissioned, it's crucial to maintain data for legal and auditing purposes while minimizing storage costs. This guide provides a comprehensive approach to data archiving with a focus on Azure storage solutions.
+
+## Best Practices for Data Archiving
+
+### Recommended Storage Account Configuration
+
+The optimal storage solution balances flexibility, support, and cost-effectiveness with the following characteristics:
+
+1. **Blob Storage Configuration**
+   - Use Archive storage tier
+   - Disable internet access
+   - Enforce authentication via Entra ID (disable access keys)
+   - Implement legal hold policies
+   - Configure multi-AZ replication in the primary region
+   - Set up single-AZ replication in a secondary region
+   - Enable object replication between regions
+   - Apply read-only lock
+
+### Sample Terraform Configuration
+
+```hcl
+resource "azurerm_storage_account" "backup_primary" {
+  name                = replace("${local.project}backupst01", "-", "")
+  resource_group_name = azurerm_resource_group.rg_itn_01.name
+  location            = local.location
+
+  account_kind             = "StorageV2"
+  account_tier             = "Standard"
+  account_replication_type = "ZRS"
+  access_tier              = "Cool"
+
+  public_network_access_enabled = false
+
+  shared_access_key_enabled       = false
+  default_to_oauth_authentication = true
+
+  blob_properties {
+    versioning_enabled       = true
+    change_feed_enabled      = true
+    last_access_time_enabled = true
+
+    delete_retention_policy {
+      days = 7
+    }
+
+    restore_policy {
+      days = 5
+    }
+
+    container_delete_retention_policy {
+      days = 10
+    }
+  }
+}
+```
+
+> **Note:** 
+> - Manually set the `access_tier` to `Archive` via Azure Portal after backup completion
+> - Set `public_network_access_enabled` to `false` only after backup is complete
+
+### Secondary Region Options
+- Germany West Central (GWC)
+- Spain Central (SPC)
+
+## Data Migration Strategies
+
+### 1. Migrating from Cosmos DB
+
+#### Tools
+- [Azure Cosmos DB Data Migration Tool (DMT)](https://github.com/AzureCosmosDB/data-migration-desktop-tool)
+
+#### Sample Migration Configuration (Cosmos DB to Blob Storage)
```

Comment:
can we add a description about the json below? ie. how is it supposed to be used?